### PR TITLE
Feature/splash footer

### PR DESCRIPTION
**Pull request by learykara**
Adds a `Footer` component that matches the pelican footer to be used on logged-out version of React apps.

### CODE REVIEW

File: src/Footer/footer.less [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  @media @nonmobile {
+    flex-basis: (@column-px * 3) + (@gutter-px * 2);
+  }
+
+  .footer-main-link {
+    color: @gray50;
+    display: block;
+    @media @nonmobile {
+      color: @white;
+      .body-text-small;
```
Comment (tholex) [inline src/Footer/footer.less]: contact-links need a hover on desktop (here)

File: src/Footer/footer.less [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  margin-right: auto;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+
+  @media @nonmobile {
+    .centered-container;
+    .grid-container;
```
Comment (tholex) [inline src/Footer/footer.less]: if you're using `display: flex` here you don't need `grid-container`

File: src/Footer/footer.less [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    text-align: left;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 80px;
+    padding-bottom: 80px;
```
Comment (tholex) [inline src/Footer/footer.less]: multiple of 15px -- 75px?

**Comment by tholex** [comment]
Watch out for http://caniuse.com/#feat=flexbox -- ~35% of IE users and ~70% of android users don't have this.

:+1: but only cause we need to move fast.

File: src/Footer/footer.less [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  .body-text-small;
+  display: none;
+  color: @blue75;
+  margin-left: 8px;
+  @media @nonmobile {
+    display: inline-block;
+  }
+}
+
+.social-links {
```
Comment (tholex) [inline src/Footer/footer.less]: `footer-social-links`? seems too generic.

**Comment by tholex** [comment]
If you're relying on flexbox this much, might as well get rid of `social-links__mobile` and __desktop. If the parent element is `display: flex;` you can use `order` to move the link to the top position on mobile instead of making two options. https://developer.mozilla.org/en-US/docs/Web/CSS/order

**Comment by tholex** [comment]
`footer-social-links` and get rid of the `__mobile / __desktop` elements and it's :+1: 

Don't make more public-facing `display: flex;` layouts though, mentor-only pages.

File: src/Footer/footer.less (hunk 3 above)
Comment (learykara) [inline src/Footer/footer.less]: copied what is on pelican currently, but I will bump it down to 75

**Comment by learykara** [comment]
closing to reopen against master

**Pull request closed without merging**